zhaquirks/tuya/ts0601_trv.py

```diff
@@ -958,7 +1140,16 @@ def __init__(self, *args, **kwargs):
                 t.int32s,
                 True,
             ),
-            ZONNSMART_WEEK_FORMAT_ATTR: ("week_format", t.uint8_t, True),
+            ZONNSMART_SCHEDULE_TYPE_ATTR: (
+                "schedule_type",
+                ZONNSMARTScheduleType,
+                True,
+            ),
```

Comment:
To generally comment on the attribute defintion:
Ideally, we already want to use the new zigpy-style `AttributeDefs` definitions, like this:
https://github.com/zigpy/zha-device-handlers/blob/ab0d248defda933019c98ddab451b5969fe99c16/zhaquirks/legrand/cable_outlet.py#L42-L59

Existing attribute defs can be extended by just inheriting that class (although that requires the base class to also use the new-style):
https://github.com/zigpy/zha-device-handlers/blob/2a39c07e48a2f3cd1956fffac009ec0c64b0f783/zhaquirks/schneiderelectric/dimmers.py#L44-L47

We should/need to convert those attributes/command definitions, but I guess it might not have to be in this PR(?), since it's already open very long. (would be nice though)

(Tuya base classes might need to be converted first for the commands. `TuyaManufClusterAttributes` doesn't have any attributes at the moment, so those could already be converted I think.)
(The advantage of the new `AttributeDefs` is also that the name/id can be accessed via `Cluster.AttributeDefs.attribute_name.id` or `.name` (if in class, self.AttributeDefs.attribute_name.id` is also enough)